app/src/interfaces/list-o2m/list-o2m.vue

```diff
@@ -111,6 +113,11 @@ const query = computed<RelationQueryMultiple>(() => {
 		return q;
 	}
 
+	if (props.sort && !relationInfo.value?.sortField) {
+		const direction = props.sort_direction ? props.sort_direction : '';
+		q.sort = [direction + props.sort];
```

Comment:
Let's align that with the manual sort string creation
```suggestion
		q.sort = [`${props.sort_direction ?? ''}${props.sort}`];
```